### ISSUE TITLE
Update 0.211.7 breaks the mod

### ISSUE THREAD

**Issue by rogerwindberg** [ontrigger/ValheimPerformanceOptimizations#36]
After today update (0.211.7) breaks the mod.

Error spam in terminal:

```
[Error  : Unity Log] MissingMethodException: string World.GetWorldSavePath()
Stack trace:
ValheimPerformanceOptimizations.Patches.MinimapGenerationPatch.MinimapTextureFilePath () (at <25e748930c1b4c01acbe72ede2c372ad>:0)
ValheimPerformanceOptimizations.Patches.MinimapGenerationPatch.Prefix (Minimap __instance) (at <25e748930c1b4c01acbe72ede2c372ad>:0)
(wrapper dynamic-method) Minimap.DMD<Minimap::GenerateWorldMap>(Minimap)
Minimap.Update () (at <a05c950b516847ac906b0be67fc12b40>:0)
```

**Comment by louis1706** [comment]
I fixed it by just removing the Patch for HeightmapColorAllocationPatch
Contact me on discord if you want the modified dll
Yamato#8987

**Comment by ontrigger** [comment]
Fixed as of 1.0.0
